src/mme/metrics.c

```diff
@@ -0,0 +1,83 @@
+#include "ogs-app.h"
+#include "mme-context.h"
+
+#include "metrics.h"
+
+typedef struct mme_metrics_spec_def_s {
+    unsigned int type;
+    const char *name;
+    const char *description;
+    int initial_val;
+    unsigned int num_labels;
+    const char **labels;
+} mme_metrics_spec_def_t;
+
+ogs_metrics_spec_t *mme_metrics_spec_global[_MME_METR_GLOB_MAX];
```

Comment:
Mark /* GLOBAL*/ section here.
Right now we only have a GLOBAL node, but other may come. Since these sections ca be lengthy, let's add something to quickly identify each, which has similar format.